.github/workflows/lint.yml

```diff
@@ -6,7 +6,7 @@ jobs:
   lint:
     runs-on: ubuntu-24.04
     container:
-      image: fedora:40
+      image: fedora:41
```

Comment:
Got a lint error due to a clang-format mismatch, bumping here since we should stick to the latest versions if possible IMO.